classes/class-uagb-front-assets.php

```diff
@@ -58,8 +58,53 @@ public static function get_instance() {
 	public function __construct() {
 		add_action( 'wp', array( $this, 'set_initial_variables' ), 99 );
 		add_action( 'wp_enqueue_scripts', array( $this, 'enqueue_asset_files' ) );
+
+		if ( is_multisite() && ! is_super_admin() && ! current_user_can( 'manage_network' ) ) {
+			add_filter( 'render_block', array( $this, 'render_icons_dynamically' ), 10, 2 );
+		}
+		
+	}
+
+	/**
+	 * Render icons dynamically for multisite.
+	 *
+	 * @param string $block_content the block data.
+	 * @since x.x.x
+	 * @return string|void
+	 */
+	public function render_icons_dynamically( $block_content ) {
+		if ( is_admin() ) {
+			return $block_content;
+		}
+
+		if ( ! empty( $block_content ) ) {
+			$pattern                = '/<span\s+class="uagb-mutisite__svg"\s+data-path="([^"]*)"(\s+data-viewbox="([^"]*)")?><\/span>/';
+			$replaced_block_content = preg_replace_callback( $pattern, array( $this, 'replace_svg' ), $block_content );
+			
+			// If the regex errored out, don't replace the $block_content.
+			$block_content = null === $replaced_block_content ? $block_content : $replaced_block_content;
```

Comment:
Any reason why we're checking specifically for `null`?

Would this code be better?

```suggestion
			$block_content = empty( $replaced_block_content ) ? $block_content : $replaced_block_content;
```

Reply:
Updated.